### DISCUSSION TITLE
My name is Being considered as a nickname when its Written in my language

### DISCUSSION THREAD

**Discussion by Mehrnikoo** [community/community#145966] [GitHub Education]
### Hi everyone,

Hello everyone,
hi the image i uploeded as for my academic affiliation has my name written in another alphabet, and when i submit it would tell me they cannot verify because it does not match the billing information, and when i change my name to my name that is written in my language it would again give me an error!!( // this feels racist as my name in my language is not a nickname!! ) 
as for the other part, i will fix it After you fixed my problem with the name

**Comment by GabJuarez** [comment]
Hi!

It can sometimes take a few days for the benefits to activate after approval. If it’s been more than a week, I recommend reaching out to GitHub Education Support.

I had a similar issue when uploading my photo. I tried uploading my college ID, but it wasn’t accepted. What worked for me was going to my college’s website and uploading a photo of my personal information instead. It doesn’t matter if your face isn’t in the photo; as long as it includes your personal information, it should be enough.


Hope you receive your benefits soon!
(I would appreciate if you mark my reply as an answer😭)

**Comment by GabJuarez** [comment]
Hi!

It can sometimes take a few days for the benefits to activate after approval. If it’s been more than a week, I recommend reaching out to GitHub Education Support.

I had a similar issue when uploading my photo. I tried uploading my college ID, but it wasn’t accepted. What worked for me was going to my college’s website and uploading a photo of my personal information instead. It doesn’t matter if your face isn’t in the photo; as long as it includes your personal information, it should be enough.


Hope you receive your benefits soon!
(I would appreciate if you mark my reply as an answer😭)
